lib/bulbs-elements/store/store.js

```diff
@@ -1,15 +1,13 @@
 import cloneObject from 'clone';
 import iterateObject from 'bulbs-elements/util/iterate-object';
-const DEBUG = false; // placeholder for real implementation
+import invariant from 'invariant';
+import { reduce } from 'lodash';
```

Comment:
👍  If we can get a verification that the production build excludes the rest of lodash. Otherwise let's get https://www.npmjs.com/package/lodash.reduce or something.